cmd/soroban-rpc/internal/methods/get_fee_stats.go

```diff
@@ -56,12 +58,18 @@ type GetFeeStatsResult struct {
 }
 
 // NewGetFeeStatsHandler returns a handler obtaining fee statistics
-func NewGetFeeStatsHandler(windows *feewindow.FeeWindows, ledgerRangeGetter LedgerRangeGetter) jrpc2.Handler {
+func NewGetFeeStatsHandler(windows *feewindow.FeeWindows, reader db.TransactionReader, logger *log.Entry) jrpc2.Handler {
```

Comment:
We should still use PRs for any add'l changes (besides upstream sync) that we want in this branch to have a proper merge log + review process.